requirements.txt

```diff
@@ -9,4 +9,6 @@ black==23.12.1
 jaxtyping==0.2.25
 beartype==0.16.4
 pre-commit==3.6.0
-isort==5.13.2
\ No newline at end of file
+isort==5.13.2
+spacy==3.7.2
+chardet==5.2.0
```

Comment:
what is it?

Reply:
`spacy` -> for LLM tasks such as analysis of tokens (part of speach POS)
`chardet` -> a dependency required for `spacy` that was not installed automatically